static/data/hierarchy_top.json

```diff
@@ -355,15 +355,24 @@
   "l": "Household",
   "t": "p",
   "e": "Household",
-  "c": 680,
+  "c": 690,
   "cd": [
    {
     "sv": [
      "Count_Household"
     ],
-    "l": "Household",
+    "l": "Count",
```

Comment:
perhaps we can call this "population" instead?

Reply:
Done